ramls/inventory/holdingsrecord.json

```diff
@@ -35,6 +36,7 @@
       "$ref" : "../uuid.json"
     },
     "permanentLocation": {
+      "description": "Inventory instances identifier",
```

Comment:
Wrong description here? 

Also, do we actually need to add these descriptions in scope of MODFEE-29?